# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/PWebRenderBridge.ipdl
# Commit: 0c25260c4edf
# Full Hash: 0c25260c4edf97fdfef11320c2a91bc0698b4c11
# Author: Alexis Beingessner <a.beingessner@gmail.com>
# Date: 2019-07-25 03:53:31
# Regressor Bug: 1547351
# File Overlap Count: 2
# Description:
#   Bug 1547351 - Automatically set content process render root r=kats
#   
#   This implements the idea of automatically setting a content proc's
#   render root based on the render root enclosing the iframe that
#   points to it. There was a bit of cleanup in here that was a bit
# ==============================================================================

diff -r f96bd2aae39c -r 0c25260c4edf gfx/layers/ipc/PWebRenderBridge.ipdl
--- a/gfx/layers/ipc/PWebRenderBridge.ipdl	Wed Jul 24 14:10:56 2019 +0000
+++ b/gfx/layers/ipc/PWebRenderBridge.ipdl	Wed Jul 24 14:10:56 2019 +0000
@@ -49,14 +49,13 @@
   async DeleteCompositorAnimations(uint64_t[] aIds);
   async SetDisplayList(RenderRootDisplayListData[] displayLists,
                        OpDestroy[] toDestroy, uint64_t fwdTransactionId, TransactionId transactionId,
-                       IdNamespace aIdNamespace, bool containsSVGGroup,
+                       bool containsSVGGroup,
                        VsyncId vsyncId, TimeStamp vsyncStartTime,
                        TimeStamp refreshStartTime, TimeStamp txnStartTime, nsCString txnURL, TimeStamp fwdTime,
                        CompositionPayload[] payloads);
-  async EmptyTransaction(FocusTarget focusTarget, uint32_t aPaintSequenceNumber,
+  async EmptyTransaction(FocusTarget focusTarget,
                          RenderRootUpdates[] renderRootUpdates,
                          OpDestroy[] toDestroy, uint64_t fwdTransactionId, TransactionId transactionId,
-                         IdNamespace aIdNamespace,
                          VsyncId vsyncId, TimeStamp vsyncStartTime,
                          TimeStamp refreshStartTime, TimeStamp txnStartTime,
                          nsCString txnURL, TimeStamp fwdTime,